### PR TITLE
Feature/full website

### PR DESCRIPTION
**Pull request by Salimer**
## What does this PR do?

This PR is raised to implement the following features 👍:

- [x] Create a website consisting of 3 pages: Home, Calculator, and Quote.
- [x] The general layout should match the provided wireframes:
Screenshot 2023-06-06 at 20 16 19

- [x] Add your own styles to improve the look and feel.
- [x] For every page, you should create a <Route>.
- [ ] Make sure that navigation links for all 3 routes are displayed on each page.
- [x] Show an error state
- [x] Use [React Router](https://reactrouter.com/) to make the routes work.

## Description of the completed tasks 🙌

- [x] Added the HTML for the structure. 
- [x] Added the SCSS file for the styling.
- [x] Added the js components for React.
- [x] Used the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <nav className="navbar">
+    <h1>Math Magicians</h1>
+    <ul className="navigation">
+      {links.map((link) => (
+        <li key={link.text}>
+          <NavLink to={link.path}>{link.text}</NavLink>
+        </li>
+      ))}
+    </ul>
+  </nav>
```
Comment (thecodechaser) [inline src/components/Navbar.js]: - [ ] Please make sure to fix the sequence of navbar options, It should be like "Home, Calculator, Quote". Please also make sure to place the border between navbar options to follow the wireframe design.


### Your design

image

### Wireframe design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Salimer👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] The issues page link in the README file is broken, Please make sure to fix this. It will the README file more professional.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salimer** [comment]
Thank you @thecodechaser for the thorough review. I addressed the following issues:
- The issues page link in the README file is broken.
- Please make sure to fix the sequence of navbar options order.
- Please also make sure to place the border between navbar options to follow the wireframe design.

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢 🟢 🟢

Hi @Salimer,

Good job working on the changes highlighted by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ The app works fine as expected with full navigation and all calculator functions 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**